build.gradle

```diff
@@ -12,7 +12,7 @@ plugins {
 }
 
 group 'uk.ac.ebi.subs'
-version '2.9.0-SNAPSHOT'
+version '2.9.2-SNAPSHOT'
```

Comment:
This is a non-breaking feature change, so you should increment the middle number. It's not a serious problem and there is no need to go and change it in this case.